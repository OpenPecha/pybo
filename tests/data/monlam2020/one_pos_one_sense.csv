edited,favorites,_id,word,result
-,-,3,ཀ་ཀ་ནཱི་ལ་,མིང་ཚིག ༡རྒྱ་གར། ཨིནྡྲ་ནཱི་ལ་དང་ནཱི་ལ་གཉིས་ལས་མདོག་ཅུང་ཟད་ནག་པའི་རྡོ་སྨན་ཁ་དོག་ཅུང་ཟད་ནག་པ། ཉི་མ་ཤར་བའི་ཕྱོགས་སུ་རང་གི་འོད་འབྱུང་ལ་ཆེ་ཆུང་མ་ངེས་པ་ཁ་དོག་སྔོ་ཞིང་ཤིན་ཏུ་དྭངས་པ་དང་དབྱིབས་སྐེད་རྔ་ཅན་ནམ། ལེབ་མོ་ངོས་མང་སོགས་སྣ་ཚོགས། ཆེ་ཆུང་མ་ངེས་ཤིང་། སྲ་ལ་མཁྲེགས་པ་ཞིག འདིས་ནད་མ་ལུས་སེལ་བར་བཤད།
