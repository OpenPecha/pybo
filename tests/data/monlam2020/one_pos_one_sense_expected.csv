ID,Form,Lemma,MonPOS,MonFeature,MonTag,POS,Feature,Morph,SenseTag,Definition,Example
1,ཀ་ཀ་ནཱི་ལ་,ཀ་ཀ་ནཱི་ལ་1,མིང་ཚིག,,༡རྒྱ་གར།,,,,ཨིནྡྲ་ནཱི་ལ་,ཨིནྡྲ་ནཱི་ལ་དང་ནཱི་ལ་གཉིས་ལས་མདོག་ཅུང་ཟད་ནག་པའི་རྡོ་སྨན་ཁ་དོག་ཅུང་ཟད་ནག་པ། ཉི་མ་ཤར་བའི་ཕྱོགས་སུ་རང་གི་འོད་འབྱུང་ལ་ཆེ་ཆུང་མ་ངེས་པ་ཁ་དོག་སྔོ་ཞིང་ཤིན་ཏུ་དྭངས་པ་དང་དབྱིབས་སྐེད་རྔ་ཅན་ནམ། ལེབ་མོ་ངོས་མང་སོགས་སྣ་ཚོགས། ཆེ་ཆུང་མ་ངེས་ཤིང་། སྲ་ལ་མཁྲེགས་པ་ཞིག འདིས་ནད་མ་ལུས་སེལ་བར་བཤད།i,
